Rakefile

```diff
@@ -89,6 +89,37 @@ task :test do
   sh 'cargo test --workspace'
 end
 
+namespace :unicode do
+  ucd_dir = 'generated/ucd'
```

Comment:
Thanks. Fixed in 314f5b2.
By the way, I end up using relative path in some places to avoid: <img width="965" alt="image" src="https://github.com/artichoke/roe/assets/12410942/b8327c83-ed21-42c0-94a3-7dcc98546079">